pkgs/applications/misc/revelation/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, pkgconfig, python2Packages, gnome2, intltool }:
+
+stdenv.mkDerivation rec {
+  name = "revelation-${version}";
+  version = "0.4.14";
+
+  src = fetchurl {
+    url = https://bitbucket.org/erikg/revelation/downloads/revelation-0.4.14.tar.bz2;
```

Comment:
You can use string interpolation to reduce the amount lines that need to be changed on updates. E.g.
`url = https://bitbucket.org/erikg/revelation/downloads/${name}.tar.bz2;`

Reply:
fixed